### DISCUSSION TITLE
Next/Previous Display doesn't work with full screen apps

### DISCUSSION THREAD

**Discussion by gabepaez** [rxhanson/Rectangle#551] [Q&A]
Nothing happens when selecting Next Display or Previous Display on a full screen app.

I love the app. Thanks for maintaining it!

macOS version: 11.5.2
Rectangle version: 0.48 (53)

**Comment by rxhanson** [comment]
Thanks for reporting. Full screened apps are treated as spaces by macOS. Unfortunately, Apple does not provide a public API for manipulating spaces, so it’s not something I plan to implement. There are ways that a couple other apps manipulate spaces, but the only ways I am aware of are quite hacky and not worth supporting in my opinion.

**Comment by gabepaez** [comment]
Bummer. There are a bunch of little annoying things about Spaces like when you switch a windowed app to a display with an active space it is trapped behind the Space. I'm sure your ability to improve upon that is probably blocked by the same lack of an API.

If you ever come up with a solution or if macOS implements an API this would be really great improvement.
